skfda/inference/anova/_anova_oneway.py

```diff
@@ -209,11 +209,14 @@ def _anova_bootstrap(
     random_state = validate_random_state(random_state)
 
     if equal_var:
-        k_est = concatenate(fd_grouped).cov().data_matrix[0, ..., 0]
+        k_est = concatenate(fd_grouped).cov().to_grid().data_matrix[0, ..., 0]
```

Comment:
Related with  #470. We need to check if it is possible to generate Gaussian data with the same covariance of a given `FDataBasis` by using the coefficients directly, instead of this costly approach.